content/source/docs/cloud/run/index.html.md

```diff
@@ -61,7 +61,7 @@ Terraform Cloud has three main workflows for managing runs, and your chosen work
 - The [API-driven run workflow](./api.html), which is more flexible but requires you to create some tooling.
 - The [CLI-driven run workflow](./cli.html), which uses Terraform's standard CLI tools to execute runs in Terraform Cloud.
 
-In more abstract terms, runs can be initiated by VCS webhooks, the manual "Start new run" action in the workspace actions menu, the standard `terraform apply` command (with the remote backend configured), and [the Runs API](../api/run.html) (or any tool that uses that API).
+You can initiate Terraform Cloud runs through the manual **Start new run** action in the workspace actions menu, VCS webhooks, the standard `terraform apply` command (with the CLI integration configured), and [the Runs API](../api/run.html) (or any tool that uses that API).
```

Comment:
This is "Start new plan" now

![image](https://user-images.githubusercontent.com/2430490/145120174-9ea183af-865c-413d-a4f9-81dbd09f8804.png)

Reply:
thank you so much for catching this!! I'm pushing up a fix.